### DISCUSSION TITLE
Looping

### DISCUSSION THREAD

**Discussion by BlackDahlia313** [stemplayer-js/stemplayer-js#4] [General]
Hey! For setting loop, I have tried to make a toggle that sets the boolean on my page and it doesn't seem to react. Is this something that has to be set for the player when it first loads atm?

**Comment by BlackDahlia313** [comment]
seems like setting it to true default doesnt loop either.

<stemplayer-js loop="true"
<stemplayer-js :loop="isLooping"

**Reply by mpcremer** [reply]
Thanks again for raising the issue. 

I've created an issue here: https://github.com/stemplayer-js/stemplayer-js/issues/5

**Comment by mpcremer** [comment]
try [3.2.1](https://www.npmjs.com/package/@stemplayer-js/stemplayer-js/v/3.2.1)
